src/main/java/seedu/address/logic/Messages.java

```diff
@@ -14,12 +14,13 @@ public class Messages {
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "Index provided is exceeds list size.";
+    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "Index provided is invalid. Index must be"
+            + "a non-zero integer no greater than the list size!";
```

Comment:
Maybe we can specify "positive integer" instead of non-zero since -1 is invalid but non-zero and not greater than the list size